### PR TITLE
doc: add meeting notes for 2018-03-14

### PR DESCRIPTION
**Pull request by MylesBorins**
refs: https://www.youtube.com/watch?v=U9xG4JPdznA
refs: https://github.com/nodejs/modules/issues/47
refs: https://docs.google.com/document/d/1fwQ_XkYyJO3p5fE2o9H9i7iWii0LB4OyQxepBlaa6-Y/edit

### CODE REVIEW

File: doc/meetings/2018-03-14.md [hunk 1]
```diff
@@ -6,27 +6,27 @@
...
-Rebecca Turner (@iarna)
-Lin Clark (@linclark)
-Guy Bedford (@guybedford)
-Bradley Farias (@bfarias)
-Jordan Harband (@ljharb)
-Hassan Sani (@inidaname)
-Justin Fagnani (@justinfagnani)
-Ben Newman (@benjamn)
-Jeremiah Senkpiel (@Fishrock123)
+Chris Dickinson (@chrisdickinson)
```
Comment (ljharb) [inline doc/meetings/2018-03-14.md]: Did you mean to add trailing spaces to all these lines?

Comment (devsnek) [reply doc/meetings/2018-03-14.md]: github needs two spaces for a newline, thats all i was adding

Comment (jkrems) [reply doc/meetings/2018-03-14.md]: Another solution without trailing whitespace: `<br/>`.

```md
Foo<br/>
Bar<br/>
```

That way it's easier to tell what's going on (imo).

Comment (devsnek) [reply doc/meetings/2018-03-14.md]: whatever works is good enough for me :P

**Pull request merged**